src/ContosoCrafts.Web.Server/Services/JsonFileProductService.cs

```diff
@@ -55,25 +61,35 @@ public async Task AddRating(string productId, int rating)
 
         public async Task<CheckoutResponse> CheckOut(IEnumerable<CartItem> Items, string callbackRoot)
         {
+            logger.LogInformation($"Count: {Items.Count()}");
+
+            List<SessionLineItemOptions> lineItems = new List<SessionLineItemOptions>();
+
+            foreach(var item in Items) {
```

Comment:
Rather than hard coding the line items, it now uses the items from the cart passed from the client.